CalendarApp/View Controllers/ComposeViewController.m

```diff
@@ -39,6 +41,11 @@ - (void)viewDidLoad {
 }
 
 - (void)setViewDate {
+    self.hasReminder = false;
+    [self.composeView.alertTimePicker setHidden:true];
+    [self.composeView.alertTimeLabel setHidden:true];
+    [self.composeView.descriptionLabelTopConstraint setConstant:20];
```

Comment:
The 20 worries me. Magic numbers should be avoided. If this is distance to top just call it that? Can it be a separate view and use the height of that to be more flexible?